client/src/components/Feed/Post.js

```diff
@@ -139,13 +141,13 @@ const Post = ({
     }
     const currentLimit = limit.current;
     limit.current = currentLimit * page;
-  };
+  });
 
   useEffect(() => {
     if (postId) {
       loadComments();
     }
-  }, []); // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loadComments, postId]); // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I guess these were added with automatic eslint fix on commit? Usually I'd suggest not including unrelated commits but since we added `--fix` as part of lint-staged (not yet working with docker) I'm okay to try to sort this out here. 

This one is creating an infinite loop so we need to use the `eslint-diable-next-line` properly. I'll commit here and keep testing.

```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```